### PR TITLE
adds an alerts sample

### PR DESCRIPTION
**Pull request by gardnerjr**
Adds a new template to azure monitor that lets you see alerts on resources (very similar to the alerts blade itself), all by querying the ARG table for alerts resources.

image

* by default, the resources are all/all/all/all for last hour.
* selecting one or more rows in the summary shows alerts for that severity
* it also uses Yves' new nosubscription template if nothing is selected:
image
* also updated the datasource docs to indicate that this datasource is "discouraged"

## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
* [x] *if updating templates, it is helpful to post a screenshot of what the changes look like.*
* [x] *if updating templates, ensure that your parameters and steps have meaningful names.*
* [x] *if adding new templates, or changing items in a gallery, it's helpful to post a screenshot of what the gallery looks like now*

### CODE REVIEW

File: Documentation/DataSources/DataSources.md [hunk 1]
```diff
@@ -53,12 +53,14 @@ To make a query control use this data source, use the _Data source_ drop down to
 
 ![A image of a workbook with ARG data](../Images/ArgDataSource.png)
 
-## Alerts
-Workbooks allow users to visualize the active alerts related to their resources. This allows the creation of reports that bring together notification data (alert) and diagnostic information (metrics, logs) into one report. This information can also be joined together to create rich reports that combines insights across these data sources. 
+## Alerts (classic)
```
Comment (ericajlin) [inline Documentation/DataSources/DataSources.md]: nit: you added classic here but the dropdown still says 'preview' which might confuse people. Maybe leave it as Alerts (preview) but highlight the suggested template

**Comment by ericajlin** [comment]
It isn't obvious that you can click on the map and have more info show up below it.
Can you add a text step below the map that indicates 'Click on the map nodes to view the alerts in the selected regions' ?
image

**Comment by yves-chan** [comment]
On Erica's note, you can also add a text step for selecting a row to show details/region

**Comment by gardnerjr** [comment]
> It isn't obvious that you can click on the map and have more info show up below it.
> Can you add a text step below the map that indicates 'Click on the map nodes to view the alerts in the selected regions' ?

i can, though i'm not a big fan of explaining everywhere that people can click things

**Pull request merged**